.github/CONTRIBUTING.md

```diff
@@ -9,13 +9,28 @@ Pathways is organized into a `monorepo`, meaning there are two projects stored w
     |src/
       |index.js
 |frontend/
-  |src/
-  |components/
-  |utils/
+  |cypress/
+  |public/
   |stories/
+  |src/
+    |components/
+    |utils/
+    |context/
+    |images/
+    |style/
+    |pages/
+      |About.jsx
+      |OccupationPage.jsx
+      |Search.jsx
 ```
 ## Running the development environment
 Both the frontend and backend environments can be run with the following terminal command (run from inside their respective root folders):
 ```bash
 npm run dev
-```
\ No newline at end of file
+```
+
+## Environment variables
+### Frontend 
+REACT_APP_SERVER gives the frontend the address for the backend
+### Backend 
+API_KEY the key for accessing the occupational maps API, BASE_URL the part of the URL use in all calls to the occupational maps API, PORT what port to use if running locally, FRONTEND the address for the frontend.
```

Comment:
maybe add here a link to the api documentation and a line to say these people should be contact to request an api key - https://occupational-maps-api.instituteforapprenticeships.org/swagger/index.html